.github/workflows/news_update.yml

```diff
@@ -16,38 +16,43 @@ jobs:
         key: ${{ secrets.data_daily_update }}
         ssh-key: ${{ secrets.data_daily_update }}
         ref: github-action
+        path: stop-covid19-sfbayarea
```

Comment:
Maybe we should make these paths shorter for easier use in other steps? e.g.

```
- uses: actions/checkout@v2
  with:
    ...
    path: site

- uses: actions/checkout@v2
  with:
    ...
    repository: sfbrigade/data-covid19-sfbayarea
    path: scraper

# Later...
- name: Install Data Scraper & Dependencies
  run: |
    cd ${GITHUB_WORKSPACE}/scraper
```

Reply:
Nice thinking! Thank you. I have fixed it.